skyportal/utils/gcn.py

```diff
@@ -68,7 +68,16 @@ def get_tags(root):
         pass
     else:
         if value == 'process.variation.burst;em.gamma':
-            yield 'GRB'
+            # Is this a GRB at all?
+            try:
+                value = root.find(".//Param[@name='GRB_Identified']").attrib['value']
+            except AttributeError:
+                yield 'GRB'
```

Comment:
If I understand what you are saying, if it finds nothing you assume it is a GRB?

Reply:
Yeah (the current behavior)